agent/engine/execcmd/manager_start_linux_test.go

```diff
@@ -160,13 +166,14 @@ func TestStartAgent(t *testing.T) {
 		} else { // No error case
 			assert.NoError(t, err, "No error was expected")
 			if !test.execEnabled {
-				assert.Nil(t, testTask.Containers[0].GetExecCommandAgentMetadata())
+				testExecCmdMD := apicontainer.ExecCommandAgentMetadata{}
+				assert.Equal(t, testExecCmdMD, testTask.Containers[0].GetExecCommandAgentMetadata())
 			} else {
-				assert.Equal(t, &apicontainer.ExecCommandAgentMetadata{
-					PID:          strconv.Itoa(testPid1),
-					DockerExecID: testDockerExecId,
-					CMD:          filepath.Join(ContainerBinDir, BinName),
-				}, testTask.Containers[0].GetExecCommandAgentMetadata())
+				execMD := testTask.Containers[0].GetExecCommandAgentMetadata()
+				assert.Equal(t, strconv.Itoa(testPid1), execMD.PID, "PID not equal")
+				assert.Equal(t, testDockerExecId, execMD.DockerExecID, "DockerExecId not equal")
+				assert.Equal(t, filepath.Join(ContainerBinDir, BinName), execMD.CMD)
+				assert.Equal(t, test.expectedStatus, execMD.Status, "Exec status not equal")
```

Comment:
can we assert here that `StartedAt` is _zero value_ for the cases where we expect `expectedStatus == apicontainerstatus.ManagedAgentStopped` and _non zero value_ for `expectedStatus == apicontainerstatus.ManagedAgentRunning`?

Reply:
sure.  I had similar time test cases in `TestGetKnownManagedAgents`.  I'll update this to be in line with those test cases.